homeassistant/components/shelly/button.py

```diff
@@ -357,6 +361,23 @@ class RpcVirtualButton(ShellyRpcAttributeEntity, ButtonEntity):
     entity_description: RpcButtonDescription
     _id: int
 
+    def __init__(
+        self,
+        coordinator: ShellyRpcCoordinator,
+        key: str,
+        attribute: str,
+        description: RpcButtonDescription,
+    ) -> None:
+        """Initialize select."""
+        super().__init__(coordinator, key, attribute, description)
+
+        if (
+            hasattr(self, "_attr_name")
+            and description.role != ROLE_GENERIC
+            and description.key != "button"
```

Comment:
I think it would be better to add translation for these two buttons (`Open` and `Close`)